projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -40,7 +43,12 @@ class Contact_Form_Endpoint extends \WP_REST_Posts_Controller {
 			'settings_url' => 'admin.php?page=zerobscrm-plugin-settings',
 		),
 		'salesforce'                        => array(
-			'type'         => 'internal',
+			'type'         => 'service',
```

Comment:
What is the difference between a service and a plugin or internal?

Reply:
Sorry, meant to add a comment on that. Internal was a mistake, so I was just fixing it here. It should be plugin or service. Service is basically any real api service (Zapier, Mailchimp, Google, etc). Plugins are plugins. 

It wasn't found/exposed before because I was hardcoding type=service in the endpoint callback before, rather than relying on the $supported_integrations array. So not DRY, and was covering up the error.